test/tslint.json

```diff
@@ -4,7 +4,25 @@
         "no-console": false,
         "no-implicit-dependencies": [
             true,
-            "dev"
-        ]
+            "dev",
+            [
+                "vscode"
+            ]
+        ],
+        "typedef": [
```

Comment:
Are we enforcing tslint in our test files too?

Reply:
We always have and this file actually _disables_ rules that I feel are unnecessary to run in test files. At the top of the file you can see it `"extends": "../tslint.json",` and all of the stuff in here is essentially set to false